### PR TITLE
Fix attention box migration to ts

### PR DESCRIPTION
**Pull request by hadasfa**
#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [Monday Design System](https://design.monday.com/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

File: src/components/AttentionBox/AttentionBox.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ interface AttentionBoxProps extends VibeComponentProps {
   // Will remove when releasing version 2 as BREAKING CHANGES
   withIconWithoutHeader?: boolean;
   /** we support 4 types of attention boxes */
-  type?: AttentionBoxTypes;
+  type?: AttentionBoxType;
   /** Icon classname for icon font or SVG Icon Component for SVG Type */
   icon?: string | React.FunctionComponent | null;
```
Comment (SergeyRoyt) [inline src/components/AttentionBox/AttentionBox.tsx:23]: I'm making icons everywhere with this type `string | React.FC<iconSubComponentProps> | null`, which I've seen somewhere. Wouldn't it be better to use it here also?

File: src/components/AttentionBox/AttentionBox.tsx [hunk 2]
```diff
@@ -138,13 +138,13 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
 };
 
 Object.assign(AttentionBox, {
-  types: AttentionBoxTypes,
+  types: AttentionBoxType,
   iconTypes: IconType
 });
 
 AttentionBox.defaultProps = {
```
Comment (SergeyRoyt) [inline src/components/AttentionBox/AttentionBox.tsx:145]: We are removing defaultProps, aren't we?

File: src/components/AttentionBox/__stories__/attentionBox.stories.mdx [hunk 3]
```diff
@@ -85,7 +85,7 @@ There are four types of attention boxes: Primary, success, danger, and dark (nat
         <AttentionBox
           title="You're doing great"
           text="Lorem Ipsum is simply dummy text of the printing and typesetting industry."
-          type={AttentionBoxTypes.SUCCESS}
+          type={AttentionBoxType.SUCCESS}
```
Comment (SergeyRoyt) [inline src/components/AttentionBox/__stories__/attentionBox.stories.mdx:88]: I guess it's better to reference `AttentionBox.types` here, no?

File: src/components/AttentionBox/AttentionBox.tsx (hunk 1 above)
Comment (hadasfa) [reply src/components/AttentionBox/AttentionBox.tsx:23]: You are right, I will change it

**Pull request merged**